### ISSUE TITLE
AlarmWrapper causing tests to fail

### ISSUE THREAD

**Issue by ajmirsky** [intelligent-agent/redeem#3]
All of the gcode tests are failing due to changes in commit a6416f3.

```
cd tests
pytest gcode/test*.py
```

```
gcode/MockPrinter.py:105: in setUpClass
    cls.R = Redeem(config_location=cfg_path)
../redeem/Redeem.py:476: in __init__
    self.printer.path_planner = PathPlanner(self.printer, pru_firmware)
../redeem/PathPlanner.py:78: in __init__
    self.__init_path_planner()
../redeem/PathPlanner.py:83: in __init_path_planner
    self.alarm_wrapper = AlarmWrapper()
../../../.pyenv/versions/2.7.6/envs/redeem/lib/python2.7/site-packages/mock/mock.py:1062: in __call__
    return _mock_self._mock_call(*args, **kwargs)
../../../.pyenv/versions/2.7.6/envs/redeem/lib/python2.7/site-packages/mock/mock.py:1121: in _mock_call
    result = next(effect)
```

**Comment by ajmirsky** [comment]
Looks like something isn't getting reset correctly in `MockPrinter.tearDownClass` since the first test that runs works ok but subsequent tests fail:
```
platform darwin -- Python 2.7.6, pytest-3.1.2, py-1.4.34, pluggy-0.4.0
rootdir: /Users/andrew/dev/redeem/tests, inifile:
collected 164 items

gcode/test_G1_G0.py .....
gcode/test_G20_G21.py EEEE
```

**Comment by ajmirsky** [comment]
this is fixed with PR #6
